cluster/examples/bridge-marker.yml

```diff
@@ -0,0 +1,78 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: bridge-marker
+  namespace: kube-system
+  labels:
+    tier: node
+    app: bridge-marker
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: bridge-marker
+    spec:
+      serviceAccountName: bridge-marker
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: bridge-marker
+        image: registry:5000/bridge-marker:latest
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+        args:
+          - -node-name
+          - $(NODE_NAME)
+        volumeMounts:
+          - name: bridge-sbin
+            mountPath: /host/sbin/ip  
+        env:
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+      volumes:
+        - name: bridge-sbin
+          hostPath:
+            path: /sbin/ip
```

Comment:
why do we mount the tool from the host and not ship it with the container image?

Reply:
This is the ip tool from the node on which the pod will run. We want to access the resources on the node, on the pod, so we have to mount the ip tool of the node to use it inside the container.

Reply:
I don't think this is how it works. That you mount the executable file doesn't mean you get access to host networking. You achieve that with hostNetwork: true in manifest. Line 17.

Reply:
Removed.
Switched to checking in /sys/class/net

Reply:
Switced to netlink